src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,50 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    public static final int CAPPACITY = 12;
+    public static final int RETURN1 = -1;
+    public static final int RETURN2 = -1;
+    final private int[] numbers = new int[CAPPACITY];
+
+
+    final static int LOOPS = 4;
+
+    final static int FIRST = -1;
+    final static int SECOND  = 11;
```

Comment:
This constant is related to capacity of numbers array. Changing only one leads to an error (inconsistency).